cmd/monaco/deploy/deploy.go

```diff
@@ -35,98 +34,99 @@ import (
 	"github.com/spf13/afero"
 )
 
-func Deploy(fs afero.Fs, deploymentManifestPath string, specificEnvironments []string, environmentGroup string,
-	specificProject []string, opts deploy.DeployConfigsOptions) error {
+func Deploy(sortedConfigs project.ConfigsPerEnvironment, environments manifest.Environments, opts deploy.DeployConfigsOptions) error {
 
-	deploymentManifestPath = filepath.Clean(deploymentManifestPath)
-	absManifestPath, err := filepath.Abs(deploymentManifestPath)
+	var deployErrs []error
+	for envName, configs := range sortedConfigs {
+		logDeploymentInfo(opts.DryRun, envName)
+		env, found := environments[envName]
+
+		if !found {
+			if opts.ContinueOnErr {
+				deployErrs = append(deployErrs, fmt.Errorf("cannot find environment `%s`", envName))
+				continue
+			} else {
+				return fmt.Errorf("cannot find environment `%s`", envName)
+			}
+		}
+
+		dtClient, err := createDynatraceClient(env, opts.DryRun)
+		if err != nil {
+			if opts.ContinueOnErr {
+				deployErrs = append(deployErrs, err)
+				continue
+			} else {
+				return err
+			}
+		}
+
+		errs := deploy.DeployConfigs(dtClient, api.NewApis(), configs, opts)
+		deployErrs = append(deployErrs, errs...)
+	}
 
-	if err != nil {
-		return fmt.Errorf("error while finding absolute path for `%s`: %w", deploymentManifestPath, err)
+	if deployErrs != nil {
+		printErrorReport(deployErrs)
+		return fmt.Errorf("errors during %s", getOperationNounForLogging(opts.DryRun))
 	}
+	log.Info("%s finished without errors", getOperationNounForLogging(opts.DryRun))
+	return nil
+}
+
+func absPath(manifestPath string) (string, error) {
```

Comment:
wwhy not move this function to `command.go` where is used (or create `util.go` file if you prefer to keep `command.go` clear)

Reply:
i've moved everything into deploy.go